src/ipyradiant/visualization/vis_tool.py

```diff
@@ -0,0 +1,114 @@
+import os
+import textwrap
+from pathlib import Path
+from rdflib import URIRef, Literal, Graph, BNode
+import traitlets as T
+from rdflib.namespace import RDF
+from pandas import DataFrame
+import ipywidgets as W
+from ipycytoscape import MutableDict, CytoscapeWidget
+
+default_edge = {"selector": "edge", "css": {"line-color": "blue"}}
+
+default_node = {"selector": "node", "css": {"background-color": "grey"}}
+
+
+class RDFVisualization(W.GridBox):
```

Comment:
consider naming this `CytoscapeVisualization` or something, in preparation for other visualizations.

if you think you've got a suitable set of base things, you could make a `BaseVisualization` and inherit from that... probably use `Widget` as the base of the base class, as not all visualizations will necessarily be boxes.